README.md

```diff
@@ -84,31 +84,16 @@ export default function MyForm() {
 
 ## File Upload example
 
-```jsx
-import { type UploadHandler } from "@remix-run/node";
-
-export const fileUploadHandler =
-  (): UploadHandler =>
-  async ({ data, filename }) => {
-    const chunks = []; 
-    for await (const chunk of data) {
-      chunks.push(chunk);
-    }
-    const buffer = Buffer.concat(chunks);
-    // If there's no filename, it's a text field and we can return the value directly
-    if (!filename) {
-      const textDecoder = new TextDecoder();
-      return textDecoder.decode(buffer);
-    }
+For more details see [File Uploads guide](https://remix.run/docs/en/main/guides/file-uploads) in Remix docs.
 
-    return new File([buffer], filename, { type: "image/jpeg" });
-  };
+```ts
+import { unstable_createMemoryUploadHandler, unstable_parseMultipartFormData, ActionFunctionArgs, json } from "@remix-run/node"; // or cloudflare/deno
 
 export const action = async ({ request }: ActionFunctionArgs) => {
   // use the upload handler to parse the file
   const formData = await unstable_parseMultipartFormData(
     request,
-    fileUploadHandler(),
+    unstable_createMemoryUploadHandler(),
```

Comment:
does this return the form data that is not a file as well?

Reply:
@MonsterDeveloper also there are conflicts!

Reply:
Yep, `parseMultipartFormData` returns everything + files.
Here's how I use it:
![CleanShot 2023-12-13 at 13 11 39](https://github.com/Code-Forge-Net/remix-hook-form/assets/16190582/f52a05c4-a9e6-4f78-95f3-deea63f164b2)

It's important to set form's `encType` to `multipart/form-data`, otherwise `parseMultipartFormData` throws an error if fed a `url-encoded` (regular) form.

Reply:
Resolved the conflict as well